python/llm/setup.py

```diff
@@ -50,8 +50,10 @@
 llm_home = os.path.join(os.path.dirname(os.path.abspath(__file__)), "src")
 github_artifact_dir = os.path.join(llm_home, '../llm-binary')
 libs_dir = os.path.join(llm_home, "ipex_llm", "libs")
+
+all_torch_version = 'torch == 2.0.1+cpu'
```

Comment:
all torch version is not a good name for this variable. Maybe change to CPU torch version